src/test_utils/es/es_test_cluster.js

```diff
@@ -0,0 +1,87 @@
+import { resolve } from 'path';
+import { esTestServerUrlParts } from '../../../test/es_test_server_url_parts';
+
+import libesvm from 'libesvm';
+
+const ESVM_DIR = resolve(__dirname, '../../../esvm/test_utils/es_test_cluster');
+
+export class EsTestCluster {
+  _freshBranches = [];
+  _directory = resolve(ESVM_DIR);
+
+  use(options = {}) {
+    const {
+      name,
+      port = esTestServerUrlParts.port,
+      log = console.log,
+      branch = 'master',
```

Comment:
This branch should come from configuration rather than being hardcoded.  Ideally using an existing configuration.